infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServerCache.java

```diff
@@ -55,12 +57,30 @@ public LogUnitServerCache(LogUnitServerConfig config, StreamLog streamLog) {
 
         MeterRegistryProvider
                 .getInstance()
-                .ifPresent(registry -> CaffeineCacheMetrics.monitor(registry, dataCache, "logunit.read_cache"));
-        MicroMeterUtils.gauge(hitRatioName, dataCache, cache -> cache.stats().hitRate());
```

Comment:
Just for my understanding : what was the bug with using the hitRate() stat?

Reply:
The rate is calculated over the life time of the cache object, but the reporting interval is per-minute. So if the cache has a high hit rate for a long period of time then a sudden drops in the hit rate wouldn't be reflected clearly. For example, if you have 100 requests per minute and the first 5 minutes the cache has a 100% hit rate, then on the 6th minute you have a 0 hit rate the calculated average would be 500 (hitRate) /600(all requests) ~ 83.3%, this can be misleading, ideally we want the reported hit rate on the 6th minute to be 0%.